locksmith/src/routes/v2/metadata.ts

```diff
@@ -47,4 +48,13 @@ router.put(
   (req, res) => metadataController.updateUserMetadata(req, res)
 )
 
+router.get(
+  '/:network/:lockAddress/data',
```

Comment:
```suggestion
  '/:network/locks/:lockAddress',
```

It's already implied the endpoint is for metadata because it's prefixed with metadata in the routes declaration.